waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -143,6 +146,18 @@ postWriteDbGeneratorActions spec dstDir = do
   dbGeneratorErrors <- maybeToList <$> generatePrismaClient spec dstDir
   pure (dbGeneratorWarnings, dbGeneratorErrors)
 
+-- | One of the checks we perform is to compare the Wasp generated schema.prisma file
+-- and the schema.prisma file in the node_modules. Prisma formats the schema in node_modules
+-- automatically, so we have to do the same to be able to compare them.
+formatPrismaSchemaFileOnDisk :: Path' Abs (Dir ProjectRootDir) -> IO ()
```

Comment:
How is it that this is new?

Reply:
I wasn't precise enough

This is new:
> Prisma formats the schema in node_modules automatically

They didn't do that and we could do our diff check without formatting our output.

Reply:
Makes sense. One question: How come we aren't comparing the ASTs? Seems more robust.

I'm guessing it was probably too much work for too litle gain.

Reply:
We are in the business of comparing checksums like in all other places in the Wasp compiler :) that's why the source files needed to be the same.